statusHandler/prometheusHandler.go

```diff
@@ -1,56 +1,56 @@
 package statusHandler
 
 import (
+	"sync"
+
 	"github.com/ElrondNetwork/elrond-go/core"
 	"github.com/prometheus/client_golang/prometheus"
-	prometheusUtils "github.com/prometheus/client_golang/prometheus/testutil"
 )
 
 // PrometheusStatusHandler will define the handler which will update prometheus metrics
 type PrometheusStatusHandler struct {
+	prometheusGaugeMetrics sync.Map
 }
 
-var prometheusGaugeMetrics map[string]prometheus.Gauge
-
 // InitMetrics will declare and init all the metrics which should be used for Prometheus
 func (psh *PrometheusStatusHandler) InitMetrics() {
-	prometheusGaugeMetrics = make(map[string]prometheus.Gauge)
+	psh.prometheusGaugeMetrics = sync.Map{}
 
 	erdSynchronizedRound := prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Please apply clean code guidance here.

Reply:
maybe init the map in another function ? single responsibility principle ?

Reply:
No, what I was thinking about was to replace calls like: 
```GO
erdSynchronizedRound := prometheus.NewGauge(prometheus.GaugeOpts{
		Name: core.MetricSynchronizedRound,
		Help: "The round where the synchronized blockchain is",
	})
	psh.prometheusGaugeMetrics.Store(core.MetricSynchronizedRound, erdSynchronizedRound)
```
with
```GO
psh.addMetric(core.MetricSynchronizedRound, "The round where the synchronized blockchain is")
```

and of course, define the func addMetric by creating a prometheus.NewGauge and then add it to the sync map.

Reply:
done